### PR TITLE
Release v0.7.4

### PR DESCRIPTION
**Pull request by taylorbot**

### CODE REVIEW

**Comment by stone-z** [comment]
Please also bump the app version if you don't mind. Looks like this was also missed for the last release
https://github.com/giantswarm/starboard-exporter/pull/217/files#diff-8b651ff5ade109d521ac0ec99d8de6047869d85138baca02acd259a8d720516dR2

**Review by marians — APPROVED** [review]

**Comment by marians** [comment]
> Please also bump the app version if you don't mind. Looks like this was also missed for the last release https://github.com/giantswarm/starboard-exporter/pull/217/files#diff-8b651ff5ade109d521ac0ec99d8de6047869d85138baca02acd259a8d720516dR2

@stone-z Sorry, I saw your comment after hitting the merge button :(

**Pull request merged**
